utils/index.js

```diff
@@ -45,3 +45,8 @@ export const truncateString = (s, maxStart, maxEnd) => {
   truncatedString += s.substr(s.length - maxEnd, s.length)
   return truncatedString
 }
+
+export const getRange = (start, end) => {
+  if(start === end) return [start]
+  return [start, ...getRange(start + 1, end)]
```

Comment:
I think this function would be better rewritten as an iterative one. Different javascript engines have different limits on the maximum level of recursion.

For example with node v16.8 I already hit the limit when using a range of 9000.

An possible iterative implementation could look something like:
```
const getRangeLoop = (start, end) => {
  if(start === end) return [start]
  return [...Array(end - start + 1).keys()].map(idx => idx + start)
}
```

I would suggest adding also some guards to ensure that end is > start or return just start in that case.

Reply:
Thanks for this, wasn't aware of it.